.github/workflows/build-image.yml

```diff
@@ -10,32 +10,81 @@ on:
     paths:
       - .github/workflows/build-image.yml
       - Dockerfile
+
 permissions:
   contents: read
+  packages: write
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
 
 jobs:
   build-image:
     runs-on: ubuntu-latest
     steps:
+      - name: Extract release name
+        env:
+          INPUT_IMAGE_NAME: ${{ secrets.IMAGE_NAME }}
+        shell: bash
+        run: |
+          if [ -z "$INPUT_IMAGE_NAME" ]; then
+              if [[ "$IMAGE_NAME" == "mastodon/mastodon" ]]; then
+                  image_name=tootsuite/mastodon
```

Comment:
Why not just use an environment variable as override? I could add it for this repository to avoid this logic.

Reply:
I'd be happy to! I didn't want to commit you to making changes to your repository to land this change, so I possibly overcomplicated it. If you're willing to set the environment variable in your repo, then I'll happily simplify this.